### PR TITLE
changed my damn mind

### PR DESCRIPTION
**Pull request by camsom**
ok. This is my final thought.

Both the constant/name/serializer need to be configured and there seems to be a fair amount of logic regarding how they are configured. This does it one place and replaces a lot of the utils logic.

### CODE REVIEW

File: bulbs/super_features/registry.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from bulbs.serializer_registry.registry import DataSerializerRegistry
+
+from .data_serializers import GuideToParentSerializer, GuideToChildSerializer
+
+
+GUIDE_TO_HOMEPAGE = 'GUIDE_TO_HOMEPAGE'
+GUIDE_TO_ENTRY = 'GUIDE_TO_ENTRY'
```
Comment (mparent61) [inline bulbs/super_features/registry.py]: Aren't these also defined in `models.py`?

Comment (benghaziboy) [inline bulbs/super_features/registry.py]: yeah. I will be moving them if we like this method.

**Comment by mparent61** [comment]
I like this. Reminds me of the DRF router approach.

Why the need for separate `registry.py` module, could you just do the registration inside `data_serializers.py`?

**Comment by benghaziboy** [comment]
Ostensibly? Might be some importing problems there///separate aims.

**Comment by camsom** [comment]
See ya in hell

**Pull request closed without merging**